### DISCUSSION TITLE
What is the chemical symbol for the element with atomic number 93 ?

### DISCUSSION THREAD

**Discussion by Sriansh-raj** [Sriansh-raj/GitHub-Achievements#33] [Galaxy Qs] [answered]
What is the chemical symbol for the element with atomic number 93 ?

**Comment by sage-etcher** [comment, marked as answer]
The element is Neptunium, and the chemical symbol is Np

**Reply by Sriansh-raj** [reply]
correct @sage-etcher 🎉 you're just one answer away from earning an achievement 😸
